app/controllers/qsets_controller.rb

```diff
@@ -10,6 +10,36 @@ def index
     end
   end
 
+  def qset_json
+    params.permit(:filter)
+    filter = params[:filter]
+    answers_json_data = []
+    $questionCount = 0
```

Comment:
What is this used for? I didn't see it referenced anywhere so I removed it from my suggested code.